packages/design_system/pubspec.yaml

```
@@ -0,0 +1,25 @@
+
+dev_dependencies:
+  build_runner: ^2.4.14
+  flutter_test:
+    sdk: flutter
+  freezed: ^2.5.8
+  json_serializable: ^6.9.0
+
+flutter:
+  uses-material-design: true
+
+resolution: workspace
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)